pypeit/scripts/compile_wvarxiv.py

```diff
@@ -0,0 +1,95 @@
+"""
+This script enables the user to convert a MasterWaveCalib wavelength solution fits file
+into a PypeIt arxiv solution that can be used with the full_template method.
+
+.. include common links, assuming primary doc root is up one directory
+.. include:: ../include/links.rst
+"""
+import time
+from pypeit import msgs
+from pypeit import par
+from pypeit import inputfiles
+from pypeit import utils
+from pypeit.scripts import scriptbase
+
+class CompileWVarxiv(scriptbase.ScriptBase):
+    """
+    A class for compiling a set of wxarxiv solutions from Identify into a single fits file.
+
+    Args:
+        wvarxiv_folder (str): Location of the WVarxiv files.
+        instrument (str): Name of the instrument (e.g., keck_lris_blue, keck_deimos, gemini_gmos_s_ham).
+        grating (str): Instrument grating name (e.g., B600, R400, 600_10000).
+        append (bool, optional): Append to an existing file for this instrument. Defaults to False.
+
+    Methods:
+        get_parser(width=None):
+            Returns an ArgumentParser object with the required command line arguments.
+        
+        main(args):
+            Main method for compiling the wxarxiv solutions into a single fits file.
+
+    Example usage:
+        parser = WvarxivCompile.get_parser()
+        args = parser.parse_args()
+        WvarxivCompile.main(args)
+    """    
+
+    @classmethod
+    def get_parser(cls, width=None):
+        parser = super().get_parser(description='Read in a set of wxarxiv solutions from Identify and compile them into a '
+                                                'single fits file to be used with the reidentify method.', width=width)
+        parser.add_argument('wvarxiv_folder', type = str, default=None, help='Location of the WVarxiv files')
+        parser.add_argument('instrument', type = str, default=None, help='Name of instrument. e.g. keck_lris_blue, keck_deimos, gemini_gmos_south_ham')
+        parser.add_argument('grating', type=str, help="Instrument grating name. E.g. b600, r400, 600_10000.")
+        parser.add_argument('--append', default=False, action='store_true', help='Append to an existing file for this instrument.')
+
+        return parser
+
+    @staticmethod
+    def main(args):
+        from astropy.table import Table, join
+        from importlib_resources import files as imres_files
+        import glob, os
+
+        # Read in the wvarxiv files
+        assert os.path.isdir(args.wvarxiv_folder), 'The wvarxiv_folder does not exist'
+        wavarxiv_files = glob.glob(args.wvarxiv_folder + '/*.fits')
```

Comment:
Use `os.path.join` instead of the `+` operator (Windows has a different syntax), but preferrably I would do (starting at line 56):
```
from pathlib import Path
wvarxiv_folder = Path(args.wvarxiv_folder).resolve()

if not wvarxiv_folder.is_dir():
    msgs.error('The wvarxiv_folder does not exist')

wvarxiv_files = sorted(wvarxiv_folder.glob('*.fits'))

if len(wvarxiv_files) == 0:
    msgs.error('No wvarxiv fits files found in the folder.')
```

Reply:
Done.